src/corrector.rb

```diff
@@ -0,0 +1,9 @@
+class Corrector
+  def initialize(name)
+    @name = name
+  end
+
+  def correct_name(name)
+    name.capitalize[1..10].strip
```

Comment:
- [ ] **`.strip`** is not needed. It is redundant since [1..10] does what it does, except better (limit of 10). Kindly remove it.

- [ ] The array(`[1..10]`) returns a name but without the first character because of **'1'** (see image below). You should use `[0..10]` instead.

  ![hector-corr](https://user-images.githubusercontent.com/36542986/147086620-865142c7-3e28-4749-84e4-6c4ada089f08.jpg)